lrc_file/LrcFile.py

```diff
@@ -192,6 +194,34 @@ def cpu_result_data(self) -> dict[str, float]:
         """
         return self._cpu_metrics
 
+    @property
+    def cpu_evaluation_data(self) -> dict[str, float]:
+        """
+            Returns CPU metrics with its values used during evaluation.
+        """
+        cpu_results = [
+            result.data["cpu"]
+            for i, result in enumerate(self._data.results)
+            if _is_cpu_measurement_result(result)
+        ]
+        evaluation_data = {}
+
+        for result in self._data.results:
+            if not isinstance(result, BaselineEvaluationResult):
+                continue
+
+            try:
+                agg_cpu_res = result.data["comparisons"][0]["current_result"].utilization
+            except AttributeError:
+                continue  # not a CPU measurement result
+
+            if agg_cpu_res in cpu_results:
+                evaluation_data[result.data["comparisons"][0]["metric_name"]] = agg_cpu_res.average
+
+                del cpu_results[cpu_results.index(agg_cpu_res)]
```

Comment:
why is this deletion here? can this condition be fullfilled twice?

Reply:
actually can, in case there are exactly same cpu measurement results, which is very unlikely to happen but as i am thinking about it now, it won't be a problem. `agg_cpu_res.average` would be same for both metrics, so it's safe to remove it.

thanks.